src/clr-core/alert/alert.base.ts

```diff
@@ -24,6 +25,8 @@ import {
 } from '@clr/core/icon-shapes';
 import { html, LitElement } from 'lit-element';
 
+ClarityIcons.addIcons(checkCircleIcon, infoCircleIcon, exclamationCircleIcon, exclamationTriangleIcon, timesIcon);
```

Comment:
The missing icon import and registering icons late was causing them to not be visible if you only imported alerts.

Reply:
Interesting. Yeah, this was the kind of stuff I had to do in the sample apps. It looks like it's going to be de-riguer for teams using the icons in a tree-shakeable manner.